interpolator/interpolator.F90

```diff
@@ -708,18 +711,14 @@ subroutine interpolator_end(clim_type)
       deallocate(clim_type%r8_type%nmon_pyear)
    end if
 endif
-if (allocated(clim_type%indexm)) deallocate(clim_type%indexm)
```

Comment:
This set of deallocation was specified twice in interpolator_end.  Removed it to make FMS more perfect.